src/index.css

```
@@ -0,0 +1,648 @@
+  background: linear-gradient(90deg, #ff6b6b 0%, #ff2b2b 40%, #ff8a00 100%);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-size: 200% 100%;
+  animation: gradientShift 6s linear infinite;
+  -webkit-text-stroke: 0.3px rgba(255, 255, 255, 0.03);
+  /* subtle edge */
+  font-weight: 800;
+  text-shadow: 0 0 rgba(255, 77, 77, 0.2);
+}
+
+@keyframes gradientShift {
```

Stylelint (17.7.0)

[error] Expected keyframe name "gradientShift" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)